.circleci/config.yml

```diff
@@ -517,6 +558,7 @@ jobs:
           # mv ~/desktop/* ~/dist/
           mv ~/ios_framework/* ~/dist/
           mv ~/android_aar/* ~/dist/
+          mv ~/js_types/* ~/dist/
```

Comment:
I noticed, I don't think these `mv` commands are doing anything. We're already outputting all the artifacts in the various build steps (the cli, ios framework, android aar, js types) to `~/dist/`.

Reply:
Guess I'm just mentioning it here to get a second opinion.

Reply:
Can't remember exactly what's going on there, but I think it's just "flattening" the dist folder... so like all the artifacts in cli are moved into the top level of dist